docs/data-builder-intro.md

```diff
@@ -69,7 +62,7 @@ patients.
     To extract data, an OpenSAFELY research study would typically use one of:
 
     * Data Builder with a dataset definition
-    * cohort-extractor with a [*study definition*]
+    * cohort-extractor with a [*study definition*](study-def.md)
```

Comment:
Not for today -- it'd be nice if we had nicer URLs.

Reply:
Agree.